docs.html

```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
                     sandbox="allow-scripts allow-same-origin">
                 </iframe>
             </section>
+
+            <!-- rating -->
+            <section id="rating">
+                <h2 class="component-title">Rating</h2>
+                <p class="component-description">Rating is used to take rating feedback from the user.
+                    Rating icons base styling is available under 'rating-icon'. The checked status can be applied using
+                    'checked' class.
+                </p>
+                <div class="rating-container">
+                    <i class="rating-icon checked fas fa-star"></i>
+                    <i class="rating-icon checked fas fa-star"></i>
+                    <i class="rating-icon checked fas fa-star"></i>
+                    <i class="rating-icon checked fas fa-star"></i>
+                    <i class="rating-icon fas fa-star"></i>
+                </div>
+                <iframe class="component-code"
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522rating-container%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+
+            <!-- textutility -->
+            <section id="textutility">
```

Comment:
`text-utility` ?
Also, no need for the comment above.

Reply:
Yeah, will consider this.